src/LeanMapper/Reflection/PropertyFactory.php

```diff
@@ -50,19 +50,19 @@ public static function createFromAnnotation($annotationType, $annotation, Entity
         $matches = [];
         $matched = preg_match(
             '~
-			^(null\|)?
-			((?:\\\\?[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)+)
-			(\[\])?
-			(\|null)?\s+
-			(\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)
-			(?:\s+=\s*(?:
-				"((?:\\\\"|[^"])+)" |  # double quoted string
-				\'((?:\\\\\'|[^\'])+)\' |  # single quoted string
-				([^ ]*))  # unquoted value
-			)?
-			(?:\s*\(([^)]+)\))?
-			(?:\s+(.*)\s*)?
-		~xi',
+            ^(null\|)?
+            ((?:\\\\?[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)+)
+            (\[\])?
+            (\|null)?\s+
+            (\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)
+            (?:\s+=\s*(?:
+                "((?:\\\\"|[^"])+)" |  # double quoted string
+                \'((?:\\\\\'|[^\'])+)\' |  # single quoted string
+                ([^ ]*))  # unquoted value
+            )?
+            (?:\s*\(([^)]+)\))?
+            (?:\s+(.*)\s*)?
+        ~xi',
```

Comment:
Všude jinde používáme taby. Nezobrazí tohle IDEčko příliš málo odsazeně? Minimálně GitHub to tak zobrazuje.